frontend/src/components/MetacodeSelect.js

```diff
@@ -13,6 +13,7 @@ import PropTypes from 'prop-types'
 
 class MetacodeSelect extends Component {
   render = () => {
+    if (!this.props.metacodeSets) return null
```

Comment:
interesting, what case does this cover? the dev server? should we console.log some instructions about how to set it up in this case?

Reply:
no, its actually when the user is logged out. it fixes a current bug where logged out user can't open the topic card